src/vmm/src/cpu_config/test_utils.rs

```diff
@@ -9,7 +9,7 @@ use crate::cpu_config::templates::CustomCpuTemplate;
 pub fn get_json_template(filename: &str) -> CustomCpuTemplate {
     let json_path = [
         env!("CARGO_MANIFEST_DIR"),
-        "../../resources/tests/static_cpu_templates",
+        "../../tests/data/static_cpu_templates",
```

Comment:
"[test: move metadata JSON files close to where they are used](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/7a1e8004cc22cd06c7bda5ab93c6206f27f211f7)" can be grouped with "[chore: move MSR data files into the test directory](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/e422717919262afc56a17086cce833f07f9404f5)".

Reply:
fixed it on my forked repo by moving "[test: move metadata JSON files close to where they are used](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/7a1e8004cc22cd06c7bda5ab93c6206f27f211f7)" soon after "[chore: move MSR data files into the test directory](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/e422717919262afc56a17086cce833f07f9404f5)": https://github.com/zulinx86/firecracker/commits/new-ci-artifacts

Reply:
also moved "[test: revamp snapshot support to Microvm](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/3197cde20ebb18fe9b6f96a66f43432d9760c06b)" and "[test: revamp .add_drive support](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/5a13e878c0c196e51d6b4188fa69c589f978d2ac)" before "[ci: regenerate new artifacts](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/e0b94bd60ae5db001b45740aea0f6f00bbf8f210)", because these two commits are not related to new CI artifacts.

Reply:
also moved "[test: default to not honor the performance baseline checks](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/43f4cda175d2a49c47bc48762272e43485a92bed)" before "[ci: regenerate new artifacts](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/e0b94bd60ae5db001b45740aea0f6f00bbf8f210)".

Reply:
also moved "[test: avoid skipped tests in test_snapshot_perf.py](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/95e4bcf017a52de39f66870c45576eeff7f87377)" before "[ci: regenerate new artifacts](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/e0b94bd60ae5db001b45740aea0f6f00bbf8f210)".